src/main/fc/fc_core.c

```diff
@@ -389,13 +402,61 @@ static bool canUpdateVTX(void)
 }
 #endif
 
+// determine if the R/P/Y stick deflection exceeds the specified limit - integer math is good enough here.
+uint8_t areSticksActive(uint8_t stickPercentLimit) {
+    uint8_t deadband;
+    uint8_t stickPercent;
+    bool result = false;
```

Comment:
Use `ret` for variable that contains return value.

Reply:
Done.